### ISSUE TITLE
[Feature Request]  .fydb file support for migration from Bluecoins

### ISSUE THREAD

**Issue by Nehemoth** [jameskokoska/Cashew#551]
**Is your feature request related to a problem? Please describe.**
No. I want to migrate to Cashew from bluecoins without losing years of data.

**Describe the solution you'd like**
Nothing happened

**Describe alternatives you've considered**
Transform the .fydb to a supported format with not avail

**Additional context**
The Bluecoins app create backup files with the .fydb file extension which its a distributed in-memory no-SQL DB.

I'm available for testing and definitely to give feedback as need it

NOTE TO THE SUBMITTER
**Please check past issues to ensure you are not duplicating feature requests. Duplicates will not be addressed again.**
Not all feature requests will be considered. This budget app was created with careful thought given to each feature's usability and prevention of feature overload. As you use the app, you will discover possibly related useful features and that may not be obvious right away. 
Feel free to email me at dapperappdeveloper@gmail.com and I'll be happy to help you!

**Comment by jameskokoska** [comment]
There are no plans to support proprietary formats. Cashew supports CSV along with provided templates which standardizes the input process.

**Comment by jameskokoska** [comment]
Related to https://github.com/jameskokoska/Cashew/issues/496

**Comment by vlkozl** [comment]
@Nehemoth I created a converter on PowerShell using AI tools to transform exported Bluecoins HTML data into a Cashew CSV format. I can share it here on GitHub if its still relevant for you.

Using the script I ran that migration for myself (I have 3 years transactions data) and that went fine, expect for Transfers, they were added like a normal transactions, but thats something that can be fixed though.
